ocw-cli/cli_app.py

```diff
@@ -599,123 +608,56 @@ def run_screen(model_datasets, models_info, observations_info,
 
              screen.addstr(6, 4, "Spatially regridding...")
              screen.refresh()
-             new_lats = np.arange(overlap_min_lat, overlap_max_lat, spatial_grid_setting)
-             new_lons = np.arange(overlap_min_lon, overlap_max_lon, spatial_grid_setting)
-             for i in range(len(obs_dataset)):
-                  obs_dataset[i] = dsp.spatial_regrid(obs_dataset[i], new_lats, new_lons)
+             if spatial_grid_option == 'Observation':
+                 new_lats = obs_dataset[0].lats
+                 new_lons = obs_dataset[0].lons
+             else:
+                 new_lats = np.arange(overlap_min_lat, overlap_max_lat, spatial_grid_setting)
+                 new_lons = np.arange(overlap_min_lon, overlap_max_lon, spatial_grid_setting)
+                 for i in range(len(obs_dataset)):
```

Comment:
spatial_grid_option == 'Observation' means that we do not regrid observation data.